Lesson4/Building/BuildingLibrary/Building.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BuildingLibrary {
+    
+    public class Building {
+        private int _id;
```

Comment:
```suggestion
        private readonly int _id;
```

Если по нему делается расчёт хэш-кода

Reply:
Да, верно... Я еще не привык задумываться о readonly* спасибо